### PR TITLE
update intro, different examples, discussion of SQL sublanguages

### PR DESCRIPTION
**Pull request by linda-seiter**
Creating new lessons for module 5 for postgreSQL, not modifying old SQLite lesson in case other courses need to use them.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,31 +2,200 @@
...
 
-## Topic 2
+- **Data** is information that can be represented in many formats
+  including numeric, textual, visual, or audio.
+- A **database** is a collection of data in a structured format.
+- A **database management systems** or **DBMS** is software that reads and writes data
+  and often provides additional features including security, performance, transactions, multi-user access, and recovery. 
+- A **relational database management system** or **RDBMS** stores data in tables, columns and rows.
+  - A table holds information about objects of a similar type.
+  - Each column within a table has a unique name.
```
Comment (kcleland0818) [inline README.md]: While we're going over basic definitions, should we also state that a column in a database table is also referred to as a field and a row is a record?

File: README.md [hunk 2]
```diff
@@ -2,31 +2,200 @@
...
+  - Each row within a table represents a unique object and must have a unique identifier, called a **primary key**.
+  - Rows among multiple tables can be related using **foreign keys**, in which one or more columns in
+    one table store the values of the primary key of another table.
+- A **query language** is a specialized programming language designed for database systems.
+  - **Structured Query Language** or **SQL** is a query language designed for relational databases.
+- **PostgreSQL** is a popular open source RDBMS.  Other RDMBS products include Oracle, MySQL, SQLite, and SQL Server.
+
+## Database Structure
+
+Relational databases store data in table that consists of
```
Comment (kcleland0818) [inline README.md]: Small typo: "Relational databases store data in table that..."

Should be "Relational databases store data in a table that..."

File: README.md [hunk 3]
```diff
@@ -2,31 +2,200 @@
...
+This type of relationship is called "one-to-many".
+
+![pet er diagram](https://curriculum-content.s3.amazonaws.com/6036/database-basics/pet_erd.png)
+
+
+Example database tables containing data for 3 owners and 9 pets could look like the following:
+
+| owner table                                                                                      | pet  table                                                                                   |
+|--------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------|
+| ![owner table](https://curriculum-content.s3.amazonaws.com/6036/database-basics/owner_table.png) | ![pet table](https://curriculum-content.s3.amazonaws.com/6036/database-basics/pet_table.png) |
```
Comment (kcleland0818) [inline README.md:68]: Lil Bub! I love Lil Bub! He was so cute. RIP. I love this example - I usually use a pet example too when describing databases.

File: README.md [hunk 4]
```diff
@@ -2,31 +2,200 @@
...
+
+### DQL (Data Query Language) 
+
+**Data Query Language** or **DQL** statements are used to perform queries on the data.
+A query selects a subset of rows and/or columns in one or more tables, and may
+perform various functions on the selected data. We will work with one DQL statement:
+
+- **SELECT** - retrieve data from the database.
+
+For example, the following SQL statement will retrieve rows
```
Comment (kcleland0818) [inline README.md]: Could we clarify this with "will retrieve all rows from the pet table" to emphasize the `*` in the SQL query?

**Review by kcleland0818 — COMMENTED** [review]
Small comments; overall looks great!

File: README.md (hunk 1 above)
Comment (linda-seiter) [reply README.md]: yes, thanks.

File: README.md (hunk 2 above)
Comment (linda-seiter) [reply README.md]: fixed.

File: README.md (hunk 3 above)
Comment (linda-seiter) [reply README.md:68]: :-)

File: README.md (hunk 4 above)
Comment (linda-seiter) [reply README.md]: updated to explain * shows all columns.

Comment (kcleland0818) [reply README.md]: Yes - I meant columns not rows lol. Thanks for understanding what I meant!

**Review by kcleland0818 — APPROVED** [review]

**Review by developerjaym — APPROVED** [review]

**Review by AlveeM — APPROVED** [review]

**Pull request merged**